config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
```

Comment:
* As mentioned by the previous CR i still experince the CORS errors in the console and I have a hint 💭 which I have tried and it worked.
* ## Cors error 👇 
![image](https://github.com/atok624/hello_rails_back_end/assets/52497246/0922e4d7-5d04-49bb-abfc-201b70c0830b)

* Kindly add this uncommented cors config below 👇 
![image](https://github.com/atok624/hello_rails_back_end/assets/52497246/0a6ee6b2-4d46-456a-be70-bb9135889067)
*